### PR TITLE
Added fetch query for filters page

### PR DESCRIPTION
**Pull request by YashKamboj**
Signed-off-by: YashKamboj <yashkamboj29@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Labeler gone wrong?

**Comment by MUzairS15** [comment]
@YashKamboj Actually, we already have subscriptions in meshery UI for `apps`, `filters` and `patterns` indirectly. What I mean is, in MeshMap we have subscriptions, so I have raised PR to incorporate same in Meshry UI (with backend as well).
Can you please incorporate the same changes when that changes gets merged??

**Comment by YashKamboj** [comment]
@MUzairS15 Would be  send a link to the pr you are talking about?

**Comment by MUzairS15** [comment]
> @MUzairS15 Would be send a link to the pr you are talking about?

Oh Sorry, https://github.com/meshery/meshery/pull/6028/files#diff-7f4f70133de77cc94ad5d59694676d2f305aed8cdd943f347f34247082d65cba

And also in this PR GraphQL query for getting catalogs is also present https://github.com/meshery/meshery/pull/6028/files#diff-b023c72e9cbb0937612593b7b30b8fcb9d965ec9c40a7e09fe3e5c432154ebb4

File: internal/graphql/resolver/schema.resolvers.go [hunk 1]
```diff
@@ -108,6 +108,10 @@ func (r *queryResolver) FetchPatterns(ctx context.Context, selector model.PageFi
 	return r.fetchPatterns(ctx, provider, selector)
 }
 
+func (r *queryResolver) FetchFilters(ctx context.Context, selector model.PageFilter) (*model.FilterPage, error) {
+	panic(fmt.Errorf("not implemented"))
```
Comment (Abhishek-kumar09) [inline internal/graphql/resolver/schema.resolvers.go:112]: panic... It will probably stop the meshery-server.

Comment (Chadha93) [reply internal/graphql/resolver/schema.resolvers.go:112]: @Abhishek-kumar09 these changes here are not required as it's covered in https://github.com/meshery/meshery/pull/6028/files#diff-b023c72e9cbb0937612593b7b30b8fcb9d965ec9c40a7e09fe3e5c432154ebb4

Comment (YashKamboj) [reply internal/graphql/resolver/schema.resolvers.go:112]: @Chadha93 Those query does not include the `page`, `page_size` and `total_count` , which are used in the `configurations` pages .

Comment (YashKamboj) [reply internal/graphql/resolver/schema.resolvers.go:112]: @Abhishek-kumar09 This was auto generated .

**Pull request closed without merging**